### PR TITLE
Replace depreciated snippets with new ones - Closes 1868

### LINKED ISSUE #1868: Replace DPoS snippets with a HelloWorld Snippets
### Description
The [Modules & Commands](https://liskhq.github.io/lisk-docs/beta/understand-blockchain/sdk/modules-commands.html) page has snippets from the DPoS module. The DPoS module no longer exists, so the snippet should be replaced with different examples.

### Affected branch / URL
`development`

### PR DESCRIPTION
**Pull request by TalhaMaliktz**
Closes #1868

### CODE REVIEW

**Review by Tschakki — APPROVED** [review]
LGTM 👍

File: docs/modules/ROOT/pages/understand-blockchain/sdk/modules-commands.adoc [hunk 1]
```diff
@@ -1378,7 +1366,7 @@ export interface CommandExecuteContext<T = undefined> {
 
 The module hooks are called in a specific order during block creation and execution.
 
-Read more about the block lifecycles in the following sections:
+Read more about the block lifecycle in the following sections:
```
Comment (chris529) [inline docs/modules/ROOT/pages/understand-blockchain/sdk/modules-commands.adoc]:
```suggestion
More information can be found regarding the block lifecycle in the following sections:
```

File: docs/modules/ROOT/pages/understand-blockchain/sdk/modules-commands.adoc [hunk 2]
```diff
@@ -347,7 +334,7 @@ Both stores are included in the xref:{url_understand_statemachine}[] of the bloc
 
 A module can define one or multiple **on-chain stores**, to store data in the blockchain, i.e. to include it in the blockchain state.
 
-For example, data like _account balances_, _delegate names_, and _multisignature keys_ are values that are stored in the on-chain module store.
+For example, data like _account balances_, _validator's names_, and _multisignature keys_ are values that are stored in the on-chain module store.
```
Comment (chris529) [inline docs/modules/ROOT/pages/understand-blockchain/sdk/modules-commands.adoc]:
```suggestion
For example, data such as _account balances_, _validator's names_, and _multisignature keys_ are values that are stored in the on-chain module store.
```

**Review by chris529 — APPROVED** [review]
LGTM! I just left 2 minor suggestions 👍

**Pull request merged**